### PR TITLE
BookStore app

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Add redux for project

- install redux 
- add books actions, reducers and action creators
- add categories action, reducer and action creator

### CODE REVIEW

File: src/redux/book/book.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (gemmen29) [inline src/redux/book/book.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions like:
```javascript
<app name>/<reducer name>/<action type>
```
for example:
```javascript
const REMOVE_BOOK = 'BookStore/book/REMOVE_BOOK';
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const GATEGORY_STATUS = 'GATEGORY_STATUS';
```
Comment (gemmen29) [inline src/redux/categories/categories.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions also here.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AhmadiGu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @AhmadiGu,

✔️ Implementation of ducks pattern

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**